dlib/cmake_utils/dlibConfig.cmake.in

```diff
@@ -1,53 +1,51 @@
-# ===================================================================================
-#  The dlib CMake configuration file
-#
-#             ** File generated automatically, do not modify **
-#
-#  Usage from an external project:
-#    In your CMakeLists.txt, add these lines:
-#
-#    find_package(dlib REQUIRED)
-#    target_link_libraries(MY_TARGET_NAME dlib::dlib)
-#
-# ===================================================================================
```

Comment:
Totally keep the instructions. People see this stuff and then go "oh right, I'll do that" rather than do a bunch of confusing hacks.